README.md

```diff
@@ -170,6 +170,7 @@ Go to [🚀 Live Demo](https://bookstore-react-redux-kappa.vercel.app/) to check
 👤 **Author**
 
 - GitHub: [@lordksix](https://github.com/lordksix)
+- GitHub: [@momo-8*](https://github.com/momo-87)
```

Comment:
Maybe there is a typo in your GitHub username?

Reply:
Thank you for your observation, there was a mistake. The required change has been made.